tests/unit/services/dialog-test.js

```diff
@@ -7,9 +7,6 @@ moduleFor('service:dialog', 'Unit | Service | dialog', {
   needs: ['component:presenter'],
   setup() {
     this.service = this.subject();
-  },
```

Comment:
I meant that the `teardown` method was removed in this commit. I pulled `commit` into the local machine. There is really no `teardown`.